index.mjs

```diff
@@ -0,0 +1,28 @@
+import mod from './index.js'
+
+export default mod;
+export const { Canvas } = mod;
+export const { Context2d } = mod;
+export const { CanvasRenderingContext2D } = mod;
+export const { CanvasGradient } = mod;
+export const { CanvasPattern } = mod;
+export const { Image } = mod;
+export const { ImageData } = mod;
+export const { PNGStream } = mod;
+export const { PDFStream } = mod;
+export const { JPEGStream } = mod;
+export const { DOMMatrix } = mod;
+export const { DOMPoint } = mod;
+export const { registerFont } = mod;
+export const { deregisterAllFonts } = mod;
+export const { parseFont } = mod;
+export const { createCanvas } = mod;
+export const { createImageData } = mod;
+export const { loadImage } = mod;
+export const { backends } = mod;
+export const { version } = mod;
+export const { cairoVersion } = mod;
+export const { jpegVersion } = mod;
+export const { gifVersion } = mod;
+export const { freetypeVersion } = mod;
+export const { rsvgVersion } = mod;
```

Comment:
```suggestion
export const {
  Canvas,
  Context2d,
  CanvasRenderingContext2D,
  CanvasGradient,
  CanvasPattern,
  Image,
  ImageData,
  PNGStream,
  DOMMatrix,
  DOMPoint,
  registerFont,
  deregisterAllFonts,
  parseFont,
  createCanvas,
  createImageData,
  loadImage,
  backends,
  version,
  cairoVersion,
  jpegVersion,
  gifVersion,
  freetypeVersion,
  rsvgVersion
} = mod;
```
Nit: Unsure about the style here, these can be narrowed down to a single destructing statement.